src/sentry/static/sentry/app/actionCreators/group.tsx

```diff
@@ -171,3 +173,199 @@ export function updateNote(
 
   return promise;
 }
+
+type ParamsType = {
+  itemIds?: Array<number> | Array<string>;
+  query?: string;
+  environment?: string | Array<string> | null;
+  project?: Array<number> | null;
+};
+
+type QueryArgs =
+  | {
+      query: string;
+      environment?: string | Array<string>;
+      project?: Array<number>;
```

Comment:
it seems that `environment` and `project` will be always the same, should we create a separate type and reuse it here?